spec/unit/type/user_spec.rb

```diff
@@ -408,7 +408,6 @@ def self.instances; []; end
       Puppet::Resource::Catalog.new :testing do |conf|
         [testuser, testrole].each { |resource| conf.add_resource resource }
       end
-      allow(Puppet::Type::User::ProviderDirectoryservice).to receive(:get_macosx_version_major).and_return("10.5")
```

Comment:
how is this change linked to 'forcelocal'?

Reply:
this is a spec file that failed on OS X because that function does not exist anymore... it's not really related